cvmfs/loader.cc

```diff
@@ -1238,7 +1238,7 @@ int FuseMain(int argc, char *argv[]) {
 
 #if CVMFS_USE_LIBFUSE != 2
   if (premount_fd >= 0) {
-    goto cleanup;
+    close(premount_fd);
```

Comment:
No, the umount does need to be done sometimes.

It should be doing a `umount2(..., MNT_DETACH)` to do a lazy unmount and not treat errors as a failure, like [libfuse](https://github.com/libfuse/libfuse/blob/fuse-3.17.3/lib/mount.c#L335).